src/gui/qgscolorwidgets.cpp

```diff
@@ -1422,13 +1438,13 @@ QgsColorSliderWidget::QgsColorSliderWidget( QWidget *parent, const ColorComponen
   mRampWidget->setColor( mCurrentColor );
   hLayout->addWidget( mRampWidget, 1 );
 
-  mSpinBox = new QSpinBox();
+  mSpinBox = new QDoubleSpinBox();
```

Comment:
This should be a QgsDoubleSpinBox, with no clear value set. Then we'd get other standard QGIS spin box behavior like the ctrl+scroll increment support.